pkg/jsonAPI/jsonapi.go

```diff
@@ -0,0 +1,514 @@
+package jsonAPI
```

Comment:
just call it `package json`.  you can rename it on import to de-conflict it with the golang json package.

or maybe just call it `api` for that matter.  This is/will be the programmatic api for sippy.

Reply:
renamed to api